week2.6.4.remember.py

```diff
@@ -0,0 +1,27 @@
+# region remember - this 2
+from PIL import Image
+
+
+def decode_message(path):
+    # Read the image from file
+    img = Image.open(path)
+
+    # Get the width and height of the image
+    width, height = img.size
+
+    # Iterate over each column of the image
+    message = ""
+    for x in range(width):
+        # Find the row number where the black pixel is located
+        for y in range(height):
+            pixel = img.getpixel((x, y))
+            if pixel == (0, 0, 0):  # black pixel found
+                # Convert the row number to its corresponding character
+                char = chr(y)
+                message += char
+                break  # move to next column

```

Comment:
1. Replace these two for loops with a list comprehention with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python

2. Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names
Try to avoid i and j as well if there is a better name available.
Goes to x and y as well